g11n-ws/modules/md-data-api-s3impl/src/main/java/com/vmware/vip/messages/data/dao/impl/S3OneComponentDaoImpl.java

```diff
@@ -41,8 +41,8 @@ public class S3OneComponentDaoImpl implements IOneComponentDao {
    @Autowired
    private S3Config config;
 
-   private static final String S3_NOT_EXIST_STR = "S3 File doesn't exist: ";
-   private static final String S3_NOT_EXIST_ERR = "File's name doesn't exist!";
+   static final String S3_NOT_EXIST_STR = "S3 File doesn't exist: ";
+   static final String S3_NOT_EXIST_ERR = "File's name doesn't exist!";
```

Comment:
S3_NOT_EXIST_ERR is not used? can remove it?

Reply:
done